frame/staking/src/pallet/mod.rs

```diff
@@ -782,6 +784,17 @@ pub mod pallet {
 			// and that MaxNominations is always greater than 1, since we count on this.
 			assert!(!T::MaxNominations::get().is_zero());
 
+			// ensure election results are always bounded with the same value
+			assert!(
+				<T::ElectionProvider as BoundedElectionProvider>::MaxWinners::get() ==
+					<T::GenesisElectionProvider as BoundedElectionProvider>::MaxWinners::get()
+			);
+			// ensure desired targets requested is always lower than max winners
+			// supported by the election provider.
+			assert!(
+				ValidatorCount::<T>::get() ==
```

Comment:
```suggestion
				ValidatorCount::<T>::get() <=
```

Why should they be equal?

Reply:
They shouldn't be. Good catch.